db/migrate/20220325172852_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
+      t.integer :postsCounter
```

Comment:
- [ ] Please update your naming according to the naming convention, your `postsCounter` should be `posts_counter`

![image](https://user-images.githubusercontent.com/75126481/160175354-0ccb7efb-1b4f-46d4-ad2b-fd7c9a807d82.png)